README.md

```diff
@@ -12,6 +12,20 @@ the same host:
 echo "API_TOKEN=<your token>" > .env
 ```
 
+* add project ID to the environment file:
```

Comment:
Please add a sub-title for the KCIDB setup as it's not required to run the pipeline.

Reply:
Okay, Is the update fine?

Reply:
Yes, thanks